<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddCabinet</class>
 <widget class="QDialog" name="DialogAddCabinet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>191</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>新增机柜</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_white" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_describe">
        <property name="text">
         <string>请填写新增机柜详细信息：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>机柜名称：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_name"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>     高度：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBox_height">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_ip">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>主要PDU IP：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_ip"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string> 设备类型：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>IP-PDU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MPDU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RPDU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NPM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ZPDU</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_deputy">
        <property name="text">
         <string> 副机：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QSpinBox" name="spinBox_deputy">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_ip_2">
        <property name="text">
         <string>备用PDU IP：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_ip_2"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string> 设备类型：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>IP-PDU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MPDU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RPDU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NPM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ZPDU</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="label_deputy_2">
        <property name="text">
         <string> 副机：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QSpinBox" name="spinBox_deputy_2">
        <property name="maximum">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QLabel" name="label_deputyDescribe">
       <property name="text">
        <string>副机（0代表：主机，1-9代表：副机序号）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(102, 199, 249);</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>spinBox_height</tabstop>
  <tabstop>lineEdit_ip</tabstop>
  <tabstop>spinBox_deputy</tabstop>
  <tabstop>lineEdit_ip_2</tabstop>
  <tabstop>spinBox_deputy_2</tabstop>
  <tabstop>pushButton_ok</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>DialogAddCabinet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
